sphinx/themes/basic/layout.html

```diff
@@ -103,7 +103,25 @@
     <link rel="stylesheet" href="{{ pathto('_static/' + style, 1) }}" type="text/css" />
     <link rel="stylesheet" href="{{ pathto('_static/pygments.css', 1) }}" type="text/css" />
     {%- for cssfile in css_files %}
-    <link rel="stylesheet" href="{{ pathto(cssfile, 1) }}" type="text/css" />
+        {%- if cssfile.filename is defined %}
+            {%- set filename = cssfile.filename %}
+        {%- else %}
+            {%- set filename = cssfile %}
+        {%- endif %}
+        {%- if cssfile.alternate is defined %}
+            {%- set alt = cssfile.alternate %}
+        {%- endif %}
+        {%- if cssfile.title is defined and cssfile.title is string %}
+            {%- if cssfile.alternate is not defined %}
+                {%- set alt = True %}
+            {%- endif %}
+    <link rel="{% if alt %}alternate {% endif %}stylesheet" href="{{ pathto(filename, 1) }}" type="text/css" title="{{ cssfile.title }}" />
+        {%- else %}
+            {%- if cssfile.alternate is not defined %}
+                {%- set alt = False %}
+            {%- endif %}
+    <link rel="{% if alt %}alternate {% endif %}stylesheet" href="{{ pathto(filename, 1) }}" type="text/css" />
+        {%- endif %}
```

Comment:
This logic shouldn't be wrote on html template because it makes difficult to read and/or understand for developers and users. LGTM except this lines.